tests/test_jwt.py

```diff
@@ -59,7 +59,7 @@ def test_sign_and_verify(self):
     self._check_sign_and_verify('privatekey.%s' % self.format)
 
   def test_sign_and_verify_from_converted_pkcs12(self):
-    """Tests that following instructions to convert from PKCS12 to PEM works."""
```

Comment:
Wait, I've without knowing been doing something wrong for years _again_? Can you link me to a source for this general frowning?